samples/react-sample-app/src/config.json

```
@@ -0,0 +1,8 @@
+{
+    "clientID": "8oY2wR89F2djzhkHP2xtVfz29Awa",
```

Let's add dummy values
```suggestion
    "clientID": <client-id>,
```

---

Resolved at https://github.com/wso2-extensions/identity-verification-onfido/pull/17